core/drand_test.go

```diff
@@ -708,52 +708,6 @@ func TestDrandPublicStream(t *testing.T) {
 	}
 }
 
-// Test if the we can correctly fetch the rounds through the local proxy
-func TestDrandPublicStreamProxy(t *testing.T) {
```

Comment:
Why removing this test ?

Reply:
I dont understand: there is still the public rand stream method using gRPC on the drand daemon [code](https://github.com/drand/drand/blob/cd378ed90c1ff2bc9c9e8a30e0bd6d783e6c84ca/core/drand_public.go#L106) - why removing a test that test this piece of code ?